lib/ansible/modules/tacp/tacp_instance.py

```diff
@@ -309,7 +530,7 @@ def instance_power_action(name, api_client, action):
         (State.PAUSED, Action.SHUTDOWN): [Action.RESUMED, Action.SHUTDOWN],
         (State.PAUSED, Action.STOPPED): [Action.STOPPED],
         (State.PAUSED, Action.RESTARTED): [Action.RESUMED, Action.RESTARTED],
-        (State.PAUSED, Action.FORCE_RESTARTED): [Action.RESUMED, Action.FORCE_RESTARTED],
+        (State.PAUSED, Action.FORCE_RESTARTED): [Action.RESUMED, Action.FORCE_RESTARTED],  # noqa
```

Comment:
There are other ways to break the line and stay within 79 char limit

```suggestion
        (State.PAUSED, Action.FORCE_RESTARTED): [
            Action.RESUMED, 
            Action.FORCE_RESTARTED
         ],
```